pravega/pravega
# Issue 6497: Adding admin cli for viewing pending events

Signed-off-by: Munish <munish.kumar2@dell.com>

**Change log description**  
Add a command ` controller-metadata get request-detail host <host-id> <request-uuid>`   in admin-cli to view pending events for a particular Controller host

**Purpose of the change**  
Fixes #[6497](https://github.com/pravega/pravega/issues/6497)

**What the code does**  
Created admin cli command controller-metadata get request-detail host <host-id> <request-uuid> to view pending events for a particular Controller host

**How to verify it**  
Navigate to pravega-admin cli.
Use command controller-metadata get request-detail host <host-id> <request-uuid> . It should print pending event for this request uuid , if there is any.
